gprofiler/metadata/metadata_collector.py

```diff
@@ -16,7 +17,7 @@ def get_static_metadata(spawn_time: float, run_args: UserArgs) -> Metadata:
     formatted_spawn_time = datetime.datetime.utcfromtimestamp(spawn_time).replace(microsecond=0).isoformat()
     static_system_metadata = get_static_system_info()
     cloud_metadata = get_static_cloud_instance_metadata(logger)
-    bigdata = get_bigdata_info()
+    bigdata = run_in_ns(["mnt"], get_bigdata_info)
```

Comment:
Btw - previously it wasn't running in host mnt NS, why changed now?

Reply:
to support cases where we see spark deployed over K8s

Reply:
To support spark over k8s